homeassistant/components/uptimerobot/switch.py

```diff
@@ -0,0 +1,76 @@
+"""UptimeRobot switch platform."""
+from __future__ import annotations
+
+from typing import Any
+
+from pyuptimerobot import UptimeRobotAuthenticationException
+
+from homeassistant.components.switch import (
+    SwitchDeviceClass,
+    SwitchEntity,
+    SwitchEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import UptimeRobotDataUpdateCoordinator
+from .const import API_ATTR_OK, DOMAIN, LOGGER
+from .entity import UptimeRobotEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the UptimeRobot switches."""
+    coordinator: UptimeRobotDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities(
+        [
+            UptimeRobotSwitch(
+                coordinator,
+                SwitchEntityDescription(
+                    key=str(monitor.id),
+                    name=f"{monitor.friendly_name} Active",
+                    device_class=SwitchDeviceClass.SWITCH,
+                ),
+                monitor=monitor,
+            )
+            for monitor in coordinator.data
+        ],
```

Comment:
async_add_entities can take a generator, no need to make this a list.

Reply:
Copied the approach from binary_sensor.py ;-)